DJHelper/DJHelper/EventPageViewController.swift

```diff
@@ -8,53 +8,75 @@
 
 import UIKit
 import MessageUI
+import CoreData
 
-class EventPageViewController: UIViewController {
+class EventPageViewController: UIViewController, UISearchBarDelegate {
 
     // MARK: - Properties
     var event: Event? {
         didSet {
+            loadViewIfNeeded()
             updateViews()
         }
     }
+
     var hostController: HostController?
 
     // If a currentHost is not passed in, hide the UI elements designed for a Host.
     var currentHost: Host?
     var eventController: EventController?
+    var songController = SongController()
```

Comment:
I'm not sure if it will matter but right now this EventPageViewController and the EventPlaylistViewController both instantiate their own SongControllers.  They probably do not need to be referring to the same one, but if they do for some reason, then we will need to instantiate it somewhere else and have both files refer to it.